src/blog/2024/11/esp32-with-node-red.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Interacting with ESP32 Using Node-RED and MQTT
+subtitle: Building IoT Flows with ESP32 and FlowFuse
+description:  Learn how to connect your ESP32 with Node-RED using MQTT in this easy-to-follow guide. Build a user-friendly dashboard with FlowFuse Dashboard to control your IoT devices remotely. Ideal for beginners and IoT hobbyists!
+date: 2024-11-14
+authors: ["sumit-shinde"]
+image: 
+keywords: esp32 with node red, node red esp32, connect esp32 to node red, esp node red, esp32 mqtt node red, esp32 node red, esp32 node red mqtt, mqtt esp32 node red, mqtt node red esp32, node red mqtt esp32, node red with esp32
+tags:
+ - node-red
+ - how-to
+---
+
+The ESP32 is a powerful microcontroller with Wi-Fi and Bluetooth, commonly used for IoT applications. This tutorial demonstrates how to set up communication between the ESP32 and Node-RED using MQTT, along with an interactive dashboard via FlowFuse for a user-friendly interface.
+
+<!--more-->
+
+## What is ESP32?
+
+![ESP32](./images/esp32.webp){data-zoomable}
+_ESP32_
+
+The ESP32 is an affordable and powerful microchip that combines Wi-Fi and Bluetooth in one small package. It's commonly used in smart devices like home automation systems, wearables, and other IoT projects. Despite its low cost (around $6), it offers strong performance, and low power consumption, and is compatible with popular platforms like Arduino. Whether you're a hobbyist or a business, the ESP32 provides great value, making it easy to create wireless devices without a big investment.
```

Comment:
Let's combine this with the sentence above, and have the video under the `more` tag.